### DISCUSSION TITLE
Can see through walls in 3d mode

### DISCUSSION THREAD

**Discussion by gera2342** [sirjuddington/SLADE#1747] [Q&A] [answered]
![20241122_02h18m23s_grim](https://github.com/user-attachments/assets/78e6a174-bbc1-4f11-ba8d-38b292fed782)

Whatever level I open in 3d mode has this. Looks like no depth testing. Is it a bug or is it a mode I can turn off?

**Comment by gera2342** [comment, marked as answer]
Must be a bug, it doesn't happen with `-DUSE_SFML_RENDERWINDOW=OFF`

**Reply by sirjuddington** [reply]
Yeah the SFML RenderWindow stuff has never really worked correctly on linux, I just removed it completely since it's no longer used in Windows either
